packages/react-native/src/primitives/Radio/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet } from 'react-native';
+
+import { RadioStyles } from './types';
+
+export const styles: RadioStyles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+  },
+  radio: {
+    height: 30,
+    width: 30,
+    borderRadius: 30,
```

Comment:
`30` is really large for a default icon on mobile, maybe we can update the size base values to:
- small: 16
- medium: 20
- large: 24

Ultimately the theme will control but think these are sensible defaults. Thoughts?

Reply:
Can we update these sizes?